es/es.go

```diff
@@ -421,7 +426,11 @@ func Run(ctx app.Context) {
 		ReturnLabels: ctx.ReturnLabels,
 		Distinct:     ctx.Options.Unique,
 	}
-	entities, err := ec.Query(ctx.Query, f)
+
+	// Create a context with the queryTimeout
+	ctxQueryTimeout, queryCancel := context.WithTimeout(context.Background(), queryTimeout)
+	defer queryCancel()
+	entities, err := ec.Query(ctxQueryTimeout, ctx.Query, f)
```

Comment:
Same as above. Shouldn't this be derived from the `ctx` parameter of `Run` instead of `context.Background`?

Reply:
The parameter `ctx` doesn't extend `context.Context` and we can not pass it into the EntityClient functions. I use the queryTimeout here since it is a parameter that can be configured via `es` and using it allows us to respect this setting.